src/common/ui-components/Search/Search.module.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../../styles/icons';
+
+.container {
+  width: 100%;
+  position: absolute;
+  background-color: white;
+  padding-left: 1rem;
+  padding-right: 1rem;
+
+  /* Hide the scrollbar, but keep scrolling enabled */
+  overflow-y: scroll;
+
+  /* Firefox */
+  scrollbar-width: none;
```

Comment:
The stylelint is supposed to complain about it, need to inestigate. Also, magic numbers